CMakeLists.txt

```diff
@@ -51,7 +51,9 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
 endif()
 
 # Install in project directory
-set(CMAKE_INSTALL_PREFIX ${PROJECT_SOURCE_DIR})
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+	set(CMAKE_INSTALL_PREFIX ${PROJECT_SOURCE_DIR})
+endif()
 # Extra CMake modules (Find*.cmake) are in CMakeModules/
```

Comment:
I was thinking that as well, but I am not sure how would that work for other platforms. In particular I am referring to this note:

> WARNING: DESTDIR may not be used on Windows because installation prefix usually contains a drive letter like in “C:/Program Files” which cannot be prepended with some other prefix.

https://cmake.org/cmake/help/v3.0/variable/CMAKE_INSTALL_PREFIX.html